src/main/java/pl/betoncraft/betonquest/compatibility/brewery/GiveBrewEvent.java

```diff
@@ -0,0 +1,57 @@
+package pl.betoncraft.betonquest.compatibility.brewery;
+
+import com.dre.brewery.BIngredients;
+import com.dre.brewery.BRecipe;
+import org.bukkit.entity.Player;
+import pl.betoncraft.betonquest.Instruction;
+import pl.betoncraft.betonquest.InstructionParseException;
+import pl.betoncraft.betonquest.QuestRuntimeException;
+import pl.betoncraft.betonquest.api.QuestEvent;
+import pl.betoncraft.betonquest.utils.PlayerConverter;
+
+public class GiveBrewEvent extends QuestEvent{
+
+    private Integer amount;
+    private Integer quality;
+    private BRecipe recipe;
+
+    public GiveBrewEvent(Instruction instruction) throws InstructionParseException {
+        super(instruction);
+
+        amount = instruction.getInt();
+
+        quality = instruction.getInt();
+
+        if(quality < 0 || quality > 10){
+            throw new InstructionParseException("Brew quality must be between 0 and 10!");
+        }
+
+        String name = instruction.next().replace("_", " ");
+
+        BRecipe recipe = null;
+        for(BRecipe r : BIngredients.recipes){
+            if(r.hasName(name)){
+                recipe = r;
+                break;
+            }
+        }
+
+        if(recipe == null){
+            throw new InstructionParseException("There is no brewing recipe with the name " + name + "!");
+        }else{
+            this.recipe = recipe;
+        }
+    }
+
+    @Override
+    public void run(String playerID) throws QuestRuntimeException {
+        Player player = PlayerConverter.getPlayer(playerID);
+
+        int remaining = amount;
+
+        while(remaining > 0){
+            player.getWorld().dropItem(player.getLocation(), recipe.create(quality));
```

Comment:
Why? I mean you could add these items to player's inventory and then drop the ones that didn't fit, but dropping them by default?

Reply:
Honestly, I was a bit lazy :D I'll fix it now